pyproject.toml

```diff
@@ -27,7 +27,7 @@ force_sort_within_sections = 'True'
 order_by_type = 'False'
 
 [tool.pytest.ini_options]
-addopts = "--color=yes"
+addopts = "-v --color=yes"
```

Comment:
Note, this is just a temporary change to make it easier to see that the `dask`-related tests are being run in CI

Reply:
```suggestion
addopts = "--color=yes"
```